src/libecalc/core/consumers/consumer_system.py

```diff
@@ -42,9 +42,11 @@
 )
 
 
-class SystemOperationalSettings(Protocol):
+class SystemComponentConditions(Protocol):
```

Comment:
If this is more or less static, it will not be a condition. In any case, name to be decided imo. Something we need to know.

Could also be possibly improved in yaml to make it easier for user to see which crossover is open and which is closed with more "verbose" and explicit yaml, if it is more or less static, then if a crossover opens or closes, that is given as an "event" instead of repeating the "final state" for all every time...

However, I didnt check usage in practice, whether it is normal to change this depending on priority...